.github/workflows/ci.yml

```
@@ -315,7 +338,7 @@ jobs:
           retention-days: 30
 
   ci-done:
-    needs: [ test, upload-coverage, asyncpg, psycopg-async, aiosqlite, sqlite3, pyright, ruff, go-test, go-lint, test-build ]
+    needs: [ test, upload-coverage, asyncpg, psycopg-async, psycopg-sync, aiosqlite, sqlite3, pyright, ruff, go-test, go-lint, test-build ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)